community/opendoas/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+printf "/chown \${BINOWN}:\${BINGRP}/d\n/chmod \${BINMODE}/d\nw\nq" |
+    ed bsd.prog.mk >/dev/null
```

Comment:
Rationale behind this?

Reply:
no real reason, pretty much just copied over from void, with execute-only permissions assigned in `post-install`.

Reply:
Would this work without the `post-install` or this `ed` call?

Reply:
yes

Reply:
So, they could both be removed?

Reply:
... done